src/home/NotificationCenter.tsx

```diff
@@ -22,6 +27,73 @@ import { Spacing } from 'src/styles/styles'
 
 type NotificationsProps = NativeStackScreenProps<StackParamList, Screens.NotificationCenter>
 
+function useCleverTapNotifications() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const messages = useSelector(cleverTapInboxMessagesSelector)
+
+  const notifications: Notification[] = []
```

Comment:
i wondered whether we should `useMemo` here, but there's not much need because the list of clevertap messages should be small right?

Reply:
good point! technically messages in CleveTap inbox can be configured to have infinite TTL. which means they will be kept forever until the user dismisses them. if the user chooses not to dismiss, they will grow up in quantity. i think it's a good idea to `useMemo` as a precaution, thanks!